pages/aer.vue

```diff
@@ -85,12 +85,12 @@ print(sim_result.get_counts(qc))</SyntaxHighlight>
 
 <script lang="ts">
 import { Component } from 'vue-property-decorator'
-import QiskitElementPage from '~/components/qiskit/QiskitElementPage.vue'
-import LegacyPresentation from '~/components/headers/LegacyPresentation.vue'
-import LegacySection from '~/components/sections/LegacySection.vue'
-import SoftwareStack from '~/components/qiskit/SoftwareStack.vue'
-import SyntaxHighlight from '~/components/qiskit/SyntaxHighlight.vue'
-import Cta from '~/components/ctas/Cta.vue'
+import QiskitElementPage from '~/components/logic/QiskitElementPage.vue'
+import LegacyPresentation from '~/components/elements/LegacyPresentation.vue'
+import LegacySection from '~/components/ui/sections/LegacySection.vue'
+import SoftwareStack from '~/components/ui/SoftwareStack.vue'
```

Comment:
Non-blocking: the `SoftwareStack` component is only used in the element pages. I'd move it under `elements/`.

Reply:
Although this component is only used in element pages,  I don't think it should be under `elements/` because is not related specifically to the domain of the elements. It is a generic component